### ISSUE TITLE
Test /firstrun test Remind Me Later

### ISSUE THREAD

**Issue by bensternthal** [mozilla/bedrock#5642]
> /firstrun test - FxA - offer "remind me later" instead of Skip on first run

Dependency
- [ ]  Become unblocked by FxA iframeless form availability #5640

**Comment by alexgibson** [comment]
Given that this page is moving in-product, I'd say we wontfix this and close the issue/bug?

**Comment by ejregithub** [comment]
Closing as invalid - this page is going in-product
